bedrock/base/templates/404.html

```diff
@@ -3,6 +3,8 @@
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 {% set_lang_files "mozorg/404" %}
 
+{% from 'macros.html' import google_custom_search with context %}
```

Comment:
This should be below the `extends` call. The `set_lang_files` above should be as well really. I know it's not your change, but we should clean that up now if you don't mind.